src/features/result-simulation/result-simulation.tsx

```diff
@@ -102,8 +102,8 @@ const ResultSimulation = (props: any) => {
       <div>
         <Breadcrumbs values={breadcrumbs} />
       </div>
-      <h1 className={styles.Title}>Simulación  con parámetros { state.is_individual ? 'individuales' : 'poblacionales'} <br/> para paciente con ci: <Link to={`/patient/${state.document_number}`} className="btn btn-primary">{state.document_number}</Link></h1>
-      <div>
+      <h1 className={styles.Title}>Simulación  con parámetros { state.is_individual ? 'individuales' : 'poblacionales'} para paciente con ci: <Link to={`/patient/${state.document_number}`} className="btn btn-primary">{state.document_number}</Link></h1>
```

Comment:
No se si me convence este titulo. Pondría algo asi:
```suggestion
      <h1 className={styles.Title}>Simulación  con parámetros { state.is_individual ? 'individuales' : 'poblacionales'} </h1>
      <h2> Paciente: <Link to={`/patient/${state.document_number}`} className="btn btn-primary">{state.document_number}</Link></h2>
```